ooxml/XSSF/Streaming/SXSSFCell.cs

```diff
@@ -196,23 +197,52 @@ public int ColumnIndex
                 return _row.GetCellIndex(this);
             }
         }
-
+        /// <summary>
+        /// Get DateTime-type cell value
+        /// </summary>
         public DateTime DateCellValue
         {
             get
             {
-                CellType cellType = _value.GetType();
-                if (cellType == CellType.Blank)
+                if (CellType == CellType.Blank)
                 {
-                    return new DateTime();
+                    throw new InvalidDataException("You cannot get a date value from a blank cell");
                 }
 
                 double value = NumericCellValue;
                 bool date1904 = Sheet.Workbook.IsDate1904();
                 return DateUtil.GetJavaDate(value, date1904);
             }
         }
-
+#if NET6_0_OR_GREATER
+        /// <summary>
+        /// Get DateOnly-type cell value
+        /// </summary>
+        public DateOnly? DateOnlyCellValue 
+        { 
+            get{
+                if (CellType == CellType.Blank||CellType == CellType.String||CellType == CellType.Boolean||CellType == CellType.Error)
```

Comment:
seems that this code gets repeated a log and only this different is the last like, whether to get DateOnly or TimeOnly, maybe could extract  helper for the beginning part?